lms/Cargo.toml

```
@@ -0,0 +1,38 @@
+[package]
+name = "lms-signature"
+version = "0.1.0"
```

We tend to bump version in a release PR/commit, so for now I'd suggest:

```suggestion
version = "0.0.0"
```